server/models/assignmentStudent.ts

```diff
@@ -0,0 +1,43 @@
+import { DataTypes } from 'sequelize';
+import sequelize from '../database/connection';
+import Students from './students';
+import Assignments from './assignments';
+
+const AssignmentStudent = sequelize.define('assignment', {
+  id: {
+    type: DataTypes.INTEGER,
+    autoIncrement: true,
+    primaryKey: true,
+  },
+  student_id: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  assignment_id: {
+    type: DataTypes.INTEGER,
+    allowNull: false,
+  },
+  isSubmitted: {
+    type: DataTypes.BOOLEAN,
+    allowNull: false,
+  },
+  material_link: {
+    type: DataTypes.TEXT,
+    allowNull: false,
+  },
```

Comment:
When adding a new assignment these values will be null, (the grade too).
So remove the `allowNull` and add a default value when you need it.